low-level/framework/base/sspl_constants.py

```diff
@@ -116,22 +116,35 @@
 salt_uniq_attr_per_node = ['cluster_id']
 salt_uniq_passwd_per_node = ['RABBITMQINGRESSPROCESSOR', 'RABBITMQEGRESSPROCESSOR', 'LOGGINGPROCESSOR']
 
-try:
-    setup_info = subprocess.Popen(['sudo', '/usr/bin/provisioner', 'get_setup_info'],
-        stdout=subprocess.PIPE).communicate()[0].decode("utf-8").rstrip()
-    setup_info = ast.literal_eval(setup_info)
-    storage_type = setup_info['storage_type'].lower()
-    server_type = setup_info['server_type'].lower()
-    logger.info(f"Storage Type : '{storage_type}'")
-    logger.info(f"Server Type '{server_type}'")
+# Provisioner API Call Keys
+NODE_KEY = 'nodes'
+SERVER_PER_NODE_KEY = 'servers_per_node'
+STORAGE_TYPE_KEY = 'storage_type'
+SERVER_TYPE_KEY = 'server_type'
+STORAGE_TYPE = 'physical'
+SERVER_TYPE = 'rbod'
```

Comment:
These are default types?

Reply:
Yes. With discussion with Indrajit. I kept the same !